src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -1,7 +1,8 @@
 """
 usage: workspace_uploader.py [-h] --workspace_id WORKSPACE_ID [--kbase_collection KBASE_COLLECTION] [--source_ver SOURCE_VER]
-                             [--root_dir ROOT_DIR] [--token_filepath TOKEN_FILEPATH] [--env {CI,NEXT,APPDEV,PROD}]
-                             [--upload_file_ext UPLOAD_FILE_EXT [UPLOAD_FILE_EXT ...]] [--workers WORKERS] [--keep_job_dir]
+                             [--root_dir ROOT_DIR] [--load_ver LOAD_VER] [--token_filepath TOKEN_FILEPATH] [--env {CI,NEXT,APPDEV,PROD}]
```

Comment:
Can we not call it `load_ver` to avoid confusion with the collections DB load ver? Maybe load_id?

Reply:
👍

Reply:
:+1: